### ISSUE TITLE
There is a bower package for this repository?

### ISSUE THREAD

**Issue by sibeliusseraphini** [john-oc/acute-select#38]
There is a bower package for this repository?

**Comment by john-oc** [comment]
No, sorry, not as yet. Bower isn't something I know very well, but I'll look into it when I have time.

**Comment by GBora** [comment]
+1 to a bower package (you'd have to remove the .sln from git and leave only the js parts)

**Comment by dmitry84** [comment]
Please check this one: https://libraries.io/bower/acute-select
